operators/pkg/controller/elasticsearch/settings/merged_config.go

```diff
@@ -10,10 +10,14 @@ import (
 	"github.com/elastic/cloud-on-k8s/operators/pkg/apis/common/v1alpha1"
 	"github.com/elastic/cloud-on-k8s/operators/pkg/controller/common/certificates"
 	common "github.com/elastic/cloud-on-k8s/operators/pkg/controller/common/settings"
-	"github.com/elastic/cloud-on-k8s/operators/pkg/controller/elasticsearch/initcontainer"
 	"github.com/elastic/cloud-on-k8s/operators/pkg/controller/elasticsearch/volume"
 )
 
+const (
+	EsContainerDataMountPath = "/usr/share/elasticsearch/data"
+	EsContainerLogsMountPath = "/usr/share/elasticsearch/logs"
```

Comment:
Not sure why these constants ended up in this file. Seems odd, even though you use them also here. Other mount paths are in `volume/names.go`

Reply:
These were specified in prepare_fs.go before and used from merged_config.go. As I couldn't have it like that due to circular dependency between settings and initcontainer I had to move it to either one of those or to a common place. I thought there might've been a specific reason it's in there, so I went with the former. Now it's in the common place :)